packages/bits/src/lib/wizard/wizard-step.component.html

```diff
@@ -5,8 +5,8 @@
         <h2 class="nui-wizard__step-title">{{title}}</h2>
         <ng-container *ngTemplateOutlet="stepTemplate"></ng-container>
     </div>
-    <nui-spinner [percent]="busyConfig.percent"
-                 [allowCancel]="busyConfig.allowCancel"
-                 [message]="busyConfig.message">
+    <nui-spinner [percent]="busyConfig.percent || 0"
+                 [allowCancel]="busyConfig.allowCancel || false"
+                 [message]="busyConfig.message || ''">
```

Comment:
It looks like spinner uses an undefined value for the `message` input in an `*ngIf` check. So, this is another place where it might be good to leave this as it was if it's not causing problems.

https://github.com/solarwinds/nova/blob/0a40bb0b469eb86c98ff824523b2d6201f1988af/packages/bits/src/lib/spinner/spinner.component.html#L29

Reply:
it's just weird that the types of these input properties do not reflect that - I should probably change those accordingly so it is obvious for consumers it can be undefined